src/app/work-item/work-item-detail/work-item-detail.component.ts

```diff
@@ -23,8 +24,7 @@ import { WorkItemService } from '../work-item.service';
 export class WorkItemDetailComponent implements OnInit {  
   workItem: WorkItem;
 
-  // TODO: These should be read from the WorkitemTypeService
-  workItemTypes = ['system.experience', 'system.feature', 'system.userstory', 'system.bug', 'system.fundamental', 'system.valueproposition'];
+  workItemTypes: WorkItemType[];
```

Comment:
@sanbornsen this PR does that. This is on the work item details page.
If you check work-item.service.ts, I have added getWorkItemTypes(). In addition to this I have created a WorkItemType object as work item type is different from work item.